### ISSUE TITLE
can you tell me some details for backPropagate function?

### ISSUE THREAD

**Issue by y1yang0** [amirjamil90/Back-Propagation-Neural-Netwrok#1]
In this function,

> def backPropagate (self, targets, N, M):

What the meaning of the constant M&N?I guess N means learning rate,but I can't understand the meaning of M, in these images I refer to
![](http://s13.sinaimg.cn/middle/60707c0f4afc63eb7b4ac&690)
![](http://s14.sinaimg.cn/middle/60707c0f4afc8dc23658d&690)
![](http://s14.sinaimg.cn/middle/60707c0f4afc913aff59d&690)
it seems that there is no constant M.

**Comment by racaljk** [comment]
might be momentum factor

**Comment by amirjamil90** [comment]
what do you want to know
